stdlib/string.gr

```diff
@@ -1867,30 +1867,49 @@ export let forEachCodePointi = (fn: (Number, Number) -> Void, str: String) => {
   void
 }
 
+@unsafe
 let trimString = (str: String, end: Bool) => {
-  let chars = explode(str), charsLength = length(str)
-  let mut i = 0, offset = 1
+  let (+) = WasmI32.add
+  let (-) = WasmI32.sub
+  let (<) = WasmI32.ltU
+  let (!=) = WasmI32.ne
+
+  let mut stringPtr = WasmI32.fromGrain(str)
+  let charsLength = WasmI32.load(stringPtr, 4n)
+  stringPtr += 8n
+  let mut i = 0n, offset = 1n
   if (end) {
-    i = charsLength - 1
-    offset = -1
+    i = charsLength - 1n
+    offset = -1n
   }
-  for (; i < charsLength && i > -1; i += offset) {
-    let currentChar = chars[i]
+  let mut count = 0n
+  for (; i < charsLength; i += offset) {
+    // Get The Character
```

Comment:
```suggestion
    // Get the byte
```

Reply:
With the explode we got actual characters, but with this approach, we're just loading bytes.